rudder_airflow_provider/hooks/rudderstack.py

```diff
@@ -47,11 +47,9 @@ def poll_for_status(self):
         while True:
             self.method = 'GET'
             resp = self.run(endpoint=status_endpoint, headers=headers).json()
-            job_status = resp['status']
-            logging.info('sync status for sourceId: %s, status: %s',
-                         self.source_id, job_status)
-
-            if job_status == STATUS_FINISHED:
+            finish_time = resp.get('finished_at') or None
```

Comment:
```suggestion
            finish_time = resp.get(FINISHED_AS) or None
```
Minor change in line 7. 
`STATUS_FINISHED = 'finished' ->  FINISHED_AT= 'finished_at'`

Reply:
Fixed